zeebe/stream-platform/src/main/java/io/camunda/zeebe/stream/impl/ProcessingScheduleServiceImpl.java

```diff
@@ -180,4 +209,25 @@ Runnable toRunnable(final Task task) {
           });
     };
   }
+
+  /** Note: this class has a natural ordering that is inconsistent with equals. */
```

Comment:
:heart:

Reply:
To be fair, IntelliJ warned me about this :D